frontend/config/environment.js

```diff
@@ -86,8 +86,12 @@ module.exports = function (environment) {
   }
 
   if (environment === "production") {
-    // No need to configure the DNS as Sentry already reads from SENTRY_DNS.
-    // https://docs.sentry.io/platforms/javascript/guides/ember/configuration/options/#dsn
+    // Whether Stage or Prod, the OIDC host and client will stay the same
+    ENV["ember-simple-auth-oidc"].host = env(
+      "OIDC_HOST",
+      "http://timed.local/auth/realms/timed/protocol/openid-connect"
```

Comment:
Not entirely sure, but the host name isn't correct IMO.

Reply:
That's just the fallback.. in case the ENV isn't provided